crates/turborepo-lib/src/query.rs

```diff
@@ -38,6 +38,23 @@ impl Query {
     }
 }
 
+#[derive(Debug, SimpleObject)]
+struct Array<T: OutputType> {
+    items: Vec<T>,
+    length: usize,
+}
+
+unsafe impl<T: Send + OutputType> Send for Array<T> {}
+unsafe impl<T: Sync + OutputType> Sync for Array<T> {}
```

Comment:
Are these necessary? Rust should be automatically implementing these if all fields are `Send`/`Sync`